executor_manager/clients/task_api_client.py

```
@@ -221,9 +225,40 @@ def _do_fetch_offline_tasks(self):
+
+            if response.status_code == 200:
+                return response.json()
+            elif response.status_code == 404:
+                logger.warning(f"Task {task_id}/{subtask_id} not found")
+                return None
+            else:
+                logger.warning(
+                    f"Failed to get task status: {response.status_code} {response.text}"
+                )
+                return None
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)